README.rst

```diff
@@ -69,10 +69,16 @@ Example usage::
 generates a file called ``migrations/20151217170514_add_id_to_users.py``
 with content::
 
+    # Uncomment should_run if this is a repeat migration
+    # def should_run(cursor):
```

Comment:
For something like a data-migration, you could write an `up` that stored the previous data in a side table, then `down` would copy that back in place. And set a `should_run` to refuse to run if the backup exists (or does something clever with timestamps for the name of the backup) Not too different than the existing `down` scripts, which tend to be frozen copies of old schema.